tests/nvmeof/workflows/ha.py

```diff
@@ -692,6 +692,10 @@ def failover(self, gateway, fail_tool):
                     LOG.info(
                         f"{list(active[0])} is new and only Active GW for failed {hostname}"
                     )
+                    active_gw = list(active[0])
```

Comment:
1) Firstly, lets add client namespace validation in `ha.run`. Lets not touch main definition.
2) Return back the active gateway node which is incharge of failed over GW in dict.

Reply:
Done.